AVRIoT.X/mcc_generated_files/credentials_storage/credentials_storage.h

```diff
@@ -29,11 +29,12 @@
 #ifndef CREDENTIALS_STORAGE_H
 #define CREDENTIALS_STORAGE_H
 
-#define MAX_WIFI_CREDENTIALS_LENGTH 31
+#include "winc/m2m/m2m_types.h"
+
 #define MAX_NTP_SERVER_LENGTH	20
 
-extern char ssid[MAX_WIFI_CREDENTIALS_LENGTH];
-extern char pass[MAX_WIFI_CREDENTIALS_LENGTH];
```

Comment:
Unnecessary limitation of the passphrase to the same length of the SSID field. Uses the right symbol and removes the limitation.